services/validation/README.md

```diff
@@ -5,6 +5,6 @@
 ##### How to?
 
 * `npm install`
-* `npm run start file1.sol metadata.json`
-    * replace `file1.sol` and `metadata.json` with relative or absolute path to your sourcefiles
-* If everything went well, proceed here [Sourcify](https://verification.komputing.org/)
\ No newline at end of file
+* `npm run start path/to/file1 path/to/file2 path/to/dir path/to/zip`
+    * the script will scan through all the provided files, directories and zips searching for metadata and the belonging source files
+* If everything went well, proceed here: [Sourcify](https://verification.komputing.org/)
```

Comment:
I think we could add some more test here. Perhaps an example run with output?

Reply:
Sure, we will update this file with more examples